docs/tutorial/01-first-steps.md

```diff
@@ -1,51 +1,59 @@
-- Resource-oriented programming, a new paradigm that pairs linear types with object capabilities to create a secure and declarative model for digital ownership
-  by ensuring that resources (and their associated assets) can only exist in one location at a time, cannot be copied, and cannot be accidentally lost or deleted
-- Built-in pre-conditions and post-conditions for functions and [transactions](../language/transactions.md)
-- The utilization of capability-based security and entitlements,
-  which enforces access control by requiring that access to objects
-  is restricted to only the owner and those who have a valid reference to the object
+:::
 
-Please see the [Cadence introduction](../index.md) for more information about the high level design of the language.
+## Objectives
 
-## What is the Flow Developer Playground?
+After completing this tutorial, you'll be able to:
 
----
+* Write, deploy, and interact with Cadence code in the Flow Playground.
+* Select and utilize accounts in the Flow Playground.
+* Run Cadence transactions from the playground.
+* Explore the contracts and storage associated with test accounts.
 
-The [Flow Playground](https://play.flow.com) includes
-an in-browser editor and Flow emulator to experiment with Flow.
-Using the Flow Playground, you can write Cadence smart contracts,
-deploy them to a local Flow emulated blockchain, and submit transactions.
 
-The Flow Playground should work with any standard web browser.
-However, we recommend using Google Chrome, as it has been primarily tested and optimized for this browser.
+:::info[Action]
 
-## Getting to know the Playground
+Instructions that require you to take action are always included in a call out box like this one.
 
-The Playground contains everything you need to get familiar
-with deploying Cadence smart contracts and interacting with transaction and scripts.
+:::
 
-The Playground comes pre-loaded with contract and transaction templates
-that correspond to each of the tutorials in the docs site.
-To load the contracts for a specific tutorial, open the desired tutorial
-in the sidebar here and click the link in the callout box at the beginning of the tutorial.
+## The Flow Developer Playground
 
-The callout boxes should look like this:
+![Flow Playground](flow-playground.png)
 
-:::tip
+The [Flow Playground] includes an in-browser editor and Flow emulator that you can use to experiment with Flow Cadence.  Using the Flow Playground, you can write Cadence smart contracts, deploy them to a local Flow emulated blockchain, and submit transactions.
+
+It has been primarily tested and optimized for Google Chrome, but other browsers will should work as well.
+
+The playground comes pre-loaded with contract and transaction templates that correspond to each of the tutorials in this series.  At the top of the page, you'll find it in a call out like this one:
+
+:::info[Action]
 
 Open the starter code for this tutorial in the Flow Playground: <br />
```

Comment:
I'm using the default at play.flow.com for here because we're actually _writing_ the contract in hello world, so I don't want to link to the end result.  

Having a different version of hello world is less than ideal, but I can't think of what else to replace it with and don't know how.  I'd figure out though if I had another idea of what should be here, but I think a slightly deeper level of a Cadence hello world, similar to what we build towards, is fine.

Reply:
Hmmm, the reason I mentioned it was because I think the cache stores your last session, so when I clicked the link another code example came up and I had to manually navigate to Hello World from the tutorial section at the top of the playground

Reply:
Yeah, that's a good point.  I've been getting confused by that myself.  Will update, thanks!